10318.xml

```diff
@@ -0,0 +1,189 @@
+        <Description><![CDATA[The Battery Level of the RCU.]]></Description>
+      </Item>
+      <Item ID="52">
+        <Name>Network Bandwidth</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Float</Type>
+        <RangeEnumeration/>
+        <Units>Mbps</Units>
+        <Description><![CDATA[Network Bandwidth of the RCU.]]></Description>
+      </Item>
+      <Item ID="53">
+        <Name>Mobile Signal</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units/>
+        <Description><![CDATA[Mobile Signal of the RCU.]]></Description>
+      </Item>
+      <Item ID="54">
+        <Name>GPS Signal</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units/>
+        <Description><![CDATA[GPS Signal of the RCU.]]></Description>
+      </Item>
+      <Item ID="55">
+        <Name>Wi-Fi Signal</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units/>
+        <Description><![CDATA[Wi-Fi Signal of the RCU.]]></Description>
+      </Item>
+      <Item ID="56">
+        <Name>UpLink Rate</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Float</Type>
+        <RangeEnumeration/>
+        <Units>Mbps</Units>
+        <Description><![CDATA[UpLink Rate of the network used by the RCU.]]></Description>
+      </Item>	  
+      <Item ID="57">
+        <Name>DownLink Rate</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Float</Type>
+        <RangeEnumeration/>
+        <Units>Mbps</Units>
+        <Description><![CDATA[DownLink Rate of the network used by the RCU.]]></Description>
+      </Item>
+      <Item ID="58">
+        <Name>Packet Loss Rate</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration>0..100</RangeEnumeration>
+        <Units>%</Units>
+        <Description><![CDATA[Packet Loss Rate in percentage.]]></Description>
+      </Item>
+      <Item ID="59">
+        <Name>Network Latency</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units>ms</Units>
```

Comment:
Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

Reply:
The unit of time, millisecond.